geopyspark-backend/util/build.sbt

```diff
@@ -0,0 +1,28 @@
+name := "util"
+
+resolvers ++= Seq(
+  "Location Tech GeoTrellis Snapshots" at "https://repo.locationtech.org/content/repositories/geotrellis-snapshots",
+  "Location Tech GeoTrellis resleases" at "https://repo.locationtech.org/content/groups/releases",
+  Resolver.mavenLocal
+)
+
+libraryDependencies ++= Seq(
+  "org.apache.spark"            %% "spark-core"            % "2.0.0" % "provided",
+  "org.locationtech.geotrellis" %% "geotrellis-s3"         % Version.geotrellis,
+  "org.locationtech.geotrellis" %% "geotrellis-spark"      % Version.geotrellis
+)
+
+assemblyMergeStrategy in assembly := {
+  case "reference.conf" => MergeStrategy.concat
+  case "application.conf" => MergeStrategy.concat
+  case "META-INF/MANIFEST.MF" => MergeStrategy.discard
+  case "META-INF\\MANIFEST.MF" => MergeStrategy.discard
+  case "META-INF/ECLIPSEF.RSA" => MergeStrategy.discard
+  case "META-INF/ECLIPSEF.SF" => MergeStrategy.discard
+  case x if x.startsWith("META-INF/services") => MergeStrategy.concat
+  case _ => MergeStrategy.first
+}
```

Comment:
This section is only needed for those projects that are going to be used as the base for assembly. The `util` is always going to be a dependency. This doesn't hurt anything but pointing it out to dis-spell ceremony.

Reply:
Ah, I see. That was most likely a copy and paste error. I'll remove that in a future update